docs/book/component-guide/orchestrators/lightning.md

```diff
@@ -0,0 +1,187 @@
+{% endhint %}
+
+
+## When to use it
+
+* You are looking for a fast and easy way to run your pipelines on GPU instances.
+
+* you're already using Lightning AI for your machine learning projects.
+
+* you want to leverage Lightning AI's managed infrastructure for running your pipelines.
+
+
+* you're looking for a solution that simplifies the deployment and scaling of your ML workflows.
+
+
+* you want to take advantage of Lightning AI's optimizations for machine learning workloads.
+
+
+## How to deploy it
+
+To use the Lightning AI orchestrator, you need to have a Lightning AI account and the necessary credentials. You don't need to deploy any additional infrastructure, as the orchestrator will use Lightning AI's managed resources.
+
+
+## How to use it
+
+To use the Lightning AI orchestrator, you need:
+
+*   The ZenML `lightning` integration installed. If you haven't done so, run
+
+    ```shell
+    zenml integration install lightning
+    ```
+
+* A [remote artifact store](../artifact-stores/artifact-stores.md) as part of your stack.
+
+
+* [Lightning AI credentials](#lightning-ai-credentials)
+
+
+### Lightning AI credentials
+
+* `LIGHTNING_USER_ID`: Your Lightning AI user ID
+
+* `LIGHTNING_API_KEY`: Your Lightning AI API key
+
+* `LIGHTNING_USERNAME`: Your Lightning AI username (optional)
+
+* `LIGHTNING_TEAMSPACE`: Your Lightning AI teamspace (optional)
+
+* `LIGHTNING_ORG`: Your Lightning AI organization (optional)
+
+
+Alternatively, you can configure these credentials when registering the orchestrator.
+
+
+We can then register the orchestrator and use it in our active stack:
+
+```shell
+zenml orchestrator register lightning_orchestrator \
+    --flavor=lightning \
+    --user_id=<YOUR_LIGHTNING_USER_ID> \
+    --api_key=<YOUR_LIGHTNING_API_KEY> \
+    --username=<YOUR_LIGHTNING_USERNAME> \
+    --teamspace=<YOUR_LIGHTNING_TEAMSPACE> \
+    --organization=<YOUR_LIGHTNING_ORGANIZATION>
+
+# Register and activate a stack with the new orchestrator
+zenml stack register lightning_stack -o lightning_orchestrator ... --set
+```
+
+You can also configure the orchestrator at pipeline level, using the `orchestrator` parameter.
+
+
+```python
+from zenml.integrations.lightning.flavors.lightning_orchestrator_flavor import LightningOrchestratorSettings
+
+
+lightning_settings = LightningOrchestratorSettings(
+    main_studio_name="my_studio",
+    machine_type="cpu",
+    async_mode=True,
+    custom_commands=["pip install -r requirements.txt", "do something else"]
+)
+
+@pipeline(
+    settings={
+        "orchestrator.lightning": lightning_settings
+    }
+)
+def my_pipeline():
+    ...
+```
+
+
+{% hint style="info" %}
+ZenML will build a wheel that includes all the files within you zenml repository and upload it to the lightning studio.
```

Comment:
this isnt happening anymore right? the `zenml init` doesn't need to be where the pipeline is and the wheel isn't created either